source/scale/elasticsearch.rst

```diff
@@ -9,17 +9,21 @@ Elasticsearch
 Elasticsearch provides enterprise-scale deployments with optimized search performance and prevents performance degradation and timeouts. The implementation uses `Elasticsearch <https://www.elastic.co>`__ v7.x as a distributed, RESTful search engine supporting highly efficient database searches in a `cluster environment </scale/high-availability-cluster.html>`__. 
 
 .. important::
-  
```

Comment:
@matthew-src - Let's keep the bulleted points as-is, and revert the non-bullet updates to this section.